HelpSource/Classes/SinOsc.schelp

```diff
@@ -7,15 +7,16 @@ categories::  UGens>Generators>Deterministic
 Description::
 
 Generates a sine wave.
-Uses a wavetable lookup oscillator with linear interpolation.
-Frequency and phase modulation are provided for audio-rate modulation.
-Technically, code::SinOsc:: uses the same implementation as  link::Classes/Osc::  except that its table is fixed to be a sine wave made of code::8192:: samples.
+Uses a wavetable oscillator with linear interpolation.
+Frequency and phase modulaton is possible on the audio rate domain.
```

Comment:
I'm afraid "is possible" should be "are possible" (i.e. this edit is grammatically inappropriate). Also I would not say "on the audio rate domain" - audio rate is not a domain. Sorry but I'd reject this edit.